### PR TITLE
feat(web): add aggregationField in ReactiveList

### PR DESCRIPTION
**Pull request by ShahAnuj2610**

### CODE REVIEW

**Review by bietkul — CHANGES_REQUESTED** [review]
@ShahAnuj2610 Why this PR have ReactiveComponent changes?

**Comment by ShahAnuj2610** [comment]
back merged, refresh once

**Review by bietkul — CHANGES_REQUESTED** [review]
@ShahAnuj2610 lgtm, you missed one change that `onData` must return the `aggregationData` key.

**Comment by ShahAnuj2610** [comment]
> @ShahAnuj2610 lgtm, you missed one change that `onData` must return the `aggregationData` key.

done

File: packages/web/src/components/result/ReactiveList.js [hunk 1]
```diff
@@ -649,27 +681,29 @@ class ReactiveList extends Component {
...
-	}
+	};
 	getComponent = () => {
-		const { error, isLoading } = this.props;
+		const { error, isLoading, aggregationData } = this.props;
 		const data = {
 			error,
 			loading: isLoading,
 			loadMore: this.loadMore,
+			aggregationData,
```
Comment (bietkul) [inline packages/web/src/components/result/ReactiveList.js]: @ShahAnuj2610 Remove it from here and merge the PR.

**Review by bietkul — CHANGES_REQUESTED** [review]

**Pull request merged**